asm/boot.s

```diff
@@ -436,13 +452,17 @@ glabel ipl3_entry // 0xA4000040
     xor   $a3, $t6, $t3
     xor   $t8, $s0, $a2
     xor   $s0, $t8, $t4
-    lui   $t3, %hi(D_B0000010)
-    lw    $t0, %lo(D_B0000010)($t3)
+#ifdef SKIP_CHECKSUM
```

Comment:
This part actually checks the checksum found in the ROM with the checksum it calculated; you can just remove this check and it still works on mupen even if you mess with the checksum in a hex editor (not integrated into config.h purely because no idea how other emulators deal with it)